Dockerfile.init

```diff
@@ -0,0 +1,29 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:latest as init-builder
+
+RUN curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+RUN chmod +x /bin/gimme
+RUN yum upgrade -y && yum install -y tar gzip git make gcc
+ENV HOME /home
+RUN /bin/gimme 1.17.9
+ENV PATH ${PATH}:/home/.gimme/versions/go1.17.9.linux.arm64/bin:/home/.gimme/versions/go1.17.9.linux.amd64/bin
```

Comment:
I've tried switching to the stable version, but this prevents me from setting the `PATH` for the go command in the Dockerfile..
Using 1.17.9 is because other Dockerfiles in our repo use this version